packages/governance/src/committeeRegistrar.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { makeNotifierKit } from '@agoric/notifier';
+import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+import { natSafeMath } from '@agoric/zoe/src/contractSupport';
+
+const { add, multiply } = natSafeMath;
+
+const start = zcf => {
+  const allQuestions = makeStore('Question');
+
+  const makeCommitteeVoterInvitation = index => {
+    const handler = voterSeat => {
+      return Far(`voter${index}`, {
+        castBallot: ballot =>
+          E(allQuestions.get(ballot.question)).submitVote(voterSeat, ballot),
```

Comment:
Why are we giving out invitations if nothing is escrowed, and this is just a way to submit votes? Why not give out a `useFacet` with the `submitVote` method on it? What is the model of who trusts who?

Reply:
Also, we do not want to encourage users to sell or otherwise pass on their invitation, right?